### PR TITLE
feat(infra): ecs migration to ec2

### LINKED ISSUE #783: [AWS] ECS FARGATE -> EC2 로 마이그레이션
### Describe the problem and solution

많은 비용 지출로 인한 마이그레이션 작업을 합니다.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/skkuding/next/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md)
- [X] Check that there isn't already an issue

### PR DESCRIPTION
**Pull request by k1g99**
Close #783 
### Description
비용문제로 ECS에서 Fargate 대신 EC2 인스턴스를 사용합니다. 

### Additional context

---

### Before submitting the PR, please make sure you do the following

- [ ] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md)
- [ ] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#pr-and-branch) and follow the [Commit Convention](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#commit-convention)
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

<a href="https://gitpod.io/#https://github.com/skkuding/codedang/pull/834"></a>

### CODE REVIEW

File: infra/modules/codedang-infra/backend/client-task-definition.tftpl [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "logConfiguration": {
       "logDriver": "awslogs",
       "options": {
-        "awslogs-group": "ecs/Codedang-Api",
+        "awslogs-group": "ecs/Codedang-Api-Client",
```
Comment (k1g99) [inline infra/modules/codedang-infra/backend/client-task-definition.tftpl:62]: client랑 admin ecs 리뷰 그룹을 구분하였습니다

**Review by dotoleeoak — APPROVED** [review]
수고하셨음다

**Comment by Goathoon** [comment]
^^ good

File: infra/modules/codedang-infra/ecs.tf [hunk 2]
```diff
@@ -34,25 +54,87 @@ data "aws_iam_policy_document" "ecs_task_role" {
...
-    resources = ["*"]
-  }
-}
+resource "aws_iam_policy" "ecs_task_execution_role" {
+  name        = "AllowEcrPull"
+  description = "Allows ECS tasks to pull images from ECR"
 
-resource "aws_iam_policy" "ses_send_email" {
-  name   = "Codedang-Api-Ses-Send-Email"
-  policy = data.aws_iam_policy_document.ses_send_email.json
```
Comment (dotoleeoak) [inline infra/modules/codedang-infra/ecs.tf:52]: 메일 권한 세팅이 사라졌구만요 😭

**Pull request merged**